qa/L0_server_launch_modes/test.sh

```diff
@@ -85,65 +85,147 @@ function run_server_launch_modes() {
 
         MODEL_ANALYZER_GLOBAL_OPTIONS="-v"
         MODEL_ANALYZER_ARGS="$MODEL_ANALYZER_BASE_ARGS `convert_gpu_array_to_flag ${gpus[@]}` -f $CONFIG_FILE"
+        MODEL_CONTROL_MODE=''
+        RELOAD_MODEL_DISABLE=''
+        MA_EXPECTED_RESULT='EP'     # Expected Pass
+
+        _do_config
+        if [ $? -ne 0 ]; then
+            break
+        fi
+    done
+}
+
+function _do_config() {
+    # Set arguments for various launch modes
+    if [ "$LAUNCH_MODE" == "remote" ]; then    
+
+        # EP = "expected pass"
+        # EF = "expected fail"
+        model_mode_combos=(
+            '--model-control-mode=explicit;;EP'
+            ';--reload-model-disable;EP'
+            '--model-control-mode=explicit;--reload-model-disable;EF' 
```

Comment:
Is the MA status code non-zero in this case? It is curious that the status code for the case below this is zero but for this case the status code is non-zero.

Reply:
yes https://gitlab-master.nvidia.com/dl/dgx/tritonmodelanalyzer/-/jobs/30549955#L6446